ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activityNumericAnswer.component.ts

```diff
@@ -60,8 +72,65 @@ export class ActivityNumericAnswer implements OnInit, OnChanges, OnDestroy {
 
     private initForm(): void {
         this.numericField = new FormControl({
-            value: _.isNumber(this.block.answer) ? this.block.answer : '',
+            value: this.mapAnswerToDisplay(this.block.answer),
             disabled: this.readonly
         }, {updateOn: 'blur'});
     }
+
+    private get isIntegerQuestion(): boolean {
+        return this.block.questionType === QuestionType.Numeric; // otherwise Question.Decimal
+    }
+
+    // e.g. for decimal: 0.71 => '0.710' (scale = 3)
+    private mapAnswerToDisplay(patchAnswer: NumericAnswerType | null): string {
+        if (patchAnswer == null) {
+            return '';
+        }
+
+        return this.isIntegerQuestion ?
+            patchAnswer.toString() :
+            this.formatDecimalAnswerToDisplay(patchAnswer);
+    }
+
+    // e.g. for decimal: '0.710' => {  // a decimal floating-point value represented by (value * 10^-scale)
+    //   "value": 710,  // the significand of a decimal number
+    //   "scale": 3     // the exponent of a decimal number
+    // }
+    private mapAnswerToPatchToServer(answerValue: string): NumericAnswerType | null {
+        if (answerValue === null) {
+            return null;
+        }
+
+        return this.isIntegerQuestion ?
+            parseInt(answerValue, 10) :
+            this.formatDecimalAnswer(answerValue);
+    }
+
+    private formatDecimalAnswer(answerValue: string): DecimalAnswer {
+        const [integerPart = '', decimalPart = ''] = answerValue.split('.');
```

Comment:
Not sure if new ESLINT rule but does it have to be initializaed like that?

Reply:
It is not eslint, just a custom logic.
I used default values ('') for cases when `integerPart ` or `decimalPart ` is `undefined` (most likely `decimalPart `, but put two initializations just for readability/consistency).
For example, if `answerValue='12345'`